src/filterDuplicate.md

```diff
@@ -0,0 +1,23 @@
+# Filter Duplicate Values
+Filters Duplicate Values From An Array.
+
+## Result
+```js
+  filterDups[1,2,2,3,4]; // '[1,2,3,4]'
```

Comment:
Invalid code, should be a proper function call

💭 Always write readable variable names